.github/workflows/golangci-lint.yml

```
@@ -35,6 +35,17 @@
       
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+          
+      - name: Build Frontend
+        run: |
+          cd frontend
+          npm install
+          npm run build
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)